notebooks/mllama-3.2/gradio_helper.py

```diff
@@ -68,7 +68,7 @@ def bot_streaming(message, history):
         prompt = processor.tokenizer.apply_chat_template(conversation, tokenize=False, add_generation_prompt=True)
         print(f"prompt is -\n{prompt}")
         image = Image.open(image)
-        inputs = processor(prompt, image, return_tensors="pt")
+        inputs = processor(image, prompt, return_tensors="pt")
```

Comment:
I suppose this fix will be more reliable:
```suggestion
        inputs = processor(images=image, text=prompt, return_tensors="pt")
```

Reply:
> I suppose this fix will be more reliable:

Agree. This one looks good. Thank you!